pages/docs/tutorials/getting-started/servers.md

```diff
@@ -0,0 +1,59 @@
+---
+title: "Servers"
+date: 2019-04-03T10:56:52+01:00
+menu:
+  docs:
+    parent: 'getting-started'
+weight: 110
+---
+
+In the previous lesson, you learned how to create the definition of a simple [Hello World application](/docs/getting-started/hello-world). Let's take it from there.
+
+In this article, you'll learn how to add `servers` to your AsyncAPI document. Adding and defining servers is useful, because it specifies where and how to connect. The connection facilitates where to send and receive messages.
+
+<CodeBlock highlightedLines={[6,7,8,9,10]}>
+{`asyncapi: 3.0.0
+info:
+  title: Hello world application
+  version: '0.1.0'
+
+servers:
+  production:
+    host: broker.mycompany.com
+    protocol: amqp
+    description: This is "My Company" broker.
+
+channels:
+  hello:
+    address: 'hello'
+    messages:
+      sayHelloMessage:
+        payload:
+          type: string
+          pattern: '^hello .+$'
+
+operations:
+  receiveHello:
+    action: 'receive'
+    channel:
+      $ref: '#/channels/hello'`}
+</CodeBlock>
```

Comment:
```suggestion
<CodeBlock highlightedLines={[6,7,8,9,10]}>
{`asyncapi: 3.0.0
info:
  title: Hello world application
  version: '0.1.0'
servers:
  production:
    host: broker.mycompany.com
    protocol: amqp
    description: This is "My Company" broker.
channels:
  hello:
    address: 'hello'
    messages:
      sayHelloMessage:
        payload:
          type: string
          pattern: '^hello .+$'
operations:
  receiveHello:
    action: 'receive'
    channel:
      $ref: '#/channels/hello'`}
</CodeBlock>
```

 CodeBlock do not like empty spaces

Reply:
> CodeBlock do not like empty spaces 

Understood :+1: 

resolved with 780b7e0